webui/react/README.md

```diff
@@ -102,6 +102,12 @@ To skip the interactive mode and run all unit tests.
 npm run test -- --watchAll=false
 ```
 
+For codecov
+
+```sh
+npm run test -- --ui --coverage --changed
+```
+
```

Comment:
```suggestion
To generate a code coverage report

```sh
npm run test:coverage
```sh

```
(the bottom backticks are labeled because the suggestion block uses triple backticks as well)

As mentioned in slack, `--changed` only works if your local worktree is dirty, otherwise vitest will not detect any tests to run.